pkg/scheduler/plugins/core/availabilityzonepriority/availability_zone_priority.go

```diff
@@ -78,11 +77,6 @@ func (pl *AvailabilityZonePriority) Score(ctx context.Context, args interface{},
 			zoneMap[zoneName] = struct{}{}
 		}
 
-		//Need to check if there is at least one pod in every zone to satisfy HA
-		if !state.SatisfyZoneAvailability(states.SchedulablePods, states) {
-			return 0, state.NewStatus(state.Unschedulable, ErrReasonNotEnoughPods)
-		}
```

Comment:
Can we add a regression test?

Reply:
Same ^. In 07ecc1f9da758883b5af8a91a2f5bda51a8692b2.